modules/mcuboot/Kconfig

```diff
@@ -44,6 +44,46 @@ config MCUBOOT_IMAGE_VERSION
 	  valuable for debugging purposes. Format: maj.min.rev+build with
 	  latter parts optional.
 
+config MCUBOOT_HARDWARE_DOWNGRADE_PREVENTION
```

Comment:
This name is unfortunate.

It can be mistaken for MCUBOOT_HW_DOWNGRADE_PREVENTION.

I suggest we name it in some way that makes it clear it is an option for configuring the MCUBoot child image.

e.g.

config CHILD_IMAGE_MCUBOOT_CONFIG_MCUBOOT_HW_DOWNGRADE_PREVENTION